firefox-ios/firefox-ios-tests/Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -388,7 +388,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
             // Workaround BB iOS13 be sure tap happens on url bar
             app.textFields.firstMatch.tap()
             app.textFields.firstMatch.tap()
-            app.textFields.firstMatch.typeText(url)
```

Comment:
This change will afffect all the tests that interact with the UrlBar.. some test type and need the field to be clear, but some others type and add text in different scenarios. I don't think we can do this change.
There is one test part of the smoketest failing in this PR and I think is related. There will be more that may fail as part of the Full Functional.
Please revert this change or create that method in the BaseTestCase to be used specifically for tests that could benefit from this, not affecting all tests and how they interact with this element.
Thanks!